tests/unit_tests/test_containers.py

```diff
@@ -52,16 +60,27 @@ def test_check_dataset_compression_above_default_threshold(self):
     def test_check_dataset_compression_below_manual_threshold(self):
         with h5py.File(name=self.file_path, mode="w") as file:
             nwb_container = self.add_dataset_to_nwb_container(file=file, gb_size=0.1)
-            assert check_dataset_compression(nwb_container=nwb_container, gb_severity_threshold=0.15) is None
+            true_output = InspectorMessage(
+                severity=Severity.LOW,
+                message="test_dataset is not compressed. Consider enabling compression when writing a dataset.",
+                importance=Importance.BEST_PRACTICE_VIOLATION,
```

Comment:
I'd rather make this a suggestion instead of using severity

Reply:
> change warning from violation to suggestion

I'm against this, suggestions are the lowest priority items but it's possible for a dataset that triggered this to be extremely large. Also, if we're already setting a lower bound to yield more accurate (and therefore fewer) trigger scenarios, then the situations where it does flag this would thus be more important.